Taarafo.Core/Migrations/20220324042442_AddPostReportModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Taarafo.Core.Migrations
+{
+    public partial class AddPostReportModel : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
+        }
+    }
```

Comment:
This file here is not needed. it just means you ran migrations with no changes. You can delete this guy